docs/resources/cluster_edge_native.md

```diff
@@ -60,7 +60,7 @@ description: |-
 Optional:
 
 - `ntp_servers` (Set of String) A list of NTP servers to be used by the cluster.
-- `ssh_keys` (Set of String) List of SSH (Secure Shell) to establish, administer, and communicate with remote clusters, `ssh_key & ssh_keys` are mutually exclusive.
+- `ssh_keys` (Set of String) List of SSH (Secure Shell) to establish, administer, and communicate with remote clusters.
```

Comment:
Will it help to callout explicitly that the keys should be `public` ? I often see this question coming from our Users. Something like, "List of *public SSH keys* to use to connect to the VM instance / node"

Reply:
@brathina-spectro sure I will add reference to public key across the  board where applicable.